modules/detect_target/detect_target_contour.py

```diff
@@ -14,19 +14,12 @@
 
 
 class DetectTargetContour(base_detect_target.BaseDetectTarget):
-    """
-    Contains the YOLOv8 model for prediction.
-    """
```

Comment:
Restore docstring and add description for contour (instead of the YOLOv8 description).